src/components/Timeline/TimelineCards/TimelineCard.tsx

```diff
@@ -29,48 +29,44 @@ const TimelineCard: FC<ITimelineCardProps> = ({ id, cardInfo }) => {
         }
     }
 
+    const timelineStyle = classNames(
+        s.cdTimelineBlock,
+        { [s.reverse]: id % 2 === 0 }
+    )
+
+    const cardContainer = classNames(
+        s.cardContainer,
+        { [s.showBack]: showBack }
+    )
+
+    const cdTimelineContent = classNames(
+        s.cdTimelineContent,
+        id % 2 === 0 ? s.arrowLeft : s.arrowRight,
+        s[cardInfo.size]
```

Comment:
Looks tricky. If you pass size that has not the same class in CSS then it wouldn't be applied and you won't catch any error about it.